src/@types/interface.ts

```diff
@@ -13,6 +13,7 @@ export interface ConfigurableOptions extends Partial<DefaultOptions> {
   customClass?: string;
   customThemeClass?: string;
   preventCloseOnClick?: boolean; // default will be false - global value for all menu items
+  normalizePosition?: boolean;
```

Comment:
Since this is optional and it has a default value, we should move it from the `ConfigurableOptions` interface to the `DefaultOptions` interface and extend 
```ts
  #defaultOptions: DefaultOptions = {
    theme: 'black',
    transitionDuration: 200,
  };
```
to 
```ts
  #defaultOptions: DefaultOptions = {
    theme: 'black',
    transitionDuration: 200,
    normalizePositon: true
  };
```